udata/core/topic/views.py

```diff
@@ -44,6 +44,7 @@ def display(topic):
     return theme.render(
         'topic/display.html',
         topic=topic,
+        datasets=[d for d in topic.datasets if hasattr(d, 'pk')],
```

Comment:
purge_datasets is run every hour in our case.
This change covers the time after deleting a dataset in a topic and before the `purge_datasets` is run which otherwise displays the empty card.
Therefore it's not crucial because it's 1h maximum (in the Datagouv project!) and we don't delete them so many times, however it's a bug display.